tests/test_citations.py

```diff
@@ -158,6 +158,34 @@ def test_citation_to_citeproc_pubmed_2():
     assert citeproc['DOI'] == '10.1161/circgenetics.115.001181'
 
 
+def test_citation_to_citeproc_pubmed_3():
+    """
+    Generated from XML returned by
+    https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pubmed&id=29028984&rettype=full
+    """
```

Comment:
Can you add a link to the issue here?

We may want to remove most of the assertions here and just do the date assertion... the question is whether now that we have some broad test coverage, do we want to start making more specific tests. I would also rename this test to `test_citation_to_citeproc_pubmed_with_numeric_month`

Reply:
Sure done in 3e3191d.  That makes sense to only test the date here.